packit/constants.py

```diff
@@ -162,8 +162,9 @@
 
 KOJI_BASEURL = "https://koji.fedoraproject.org/kojihub"
 
-CHROOT_SPECIFIC_COPR_CONFIGURATION = (
-    "additional_packages",
-    "additional_repos",
-    "additional_modules",
-)
+# key name -> default
+CHROOT_SPECIFIC_COPR_CONFIGURATION = {
+    "additional_packages": [],
+    "additional_repos": [],
+    "additional_modules": "",
```

Comment:
:man_facepalming:

Reply:
Copr stores it as a string in the db (and probably passes it as a string further, hence this syntax).

Not perfect, but good enough I'd say.

Reply:
I added a comment about this in the code.

Reply:
I see, it was not a reaction to your code.